### PR TITLE
reactive form for Angular

### PR DESCRIPTION
**Pull request by lvncnt**
#3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -344,7 +344,28 @@ export class UserPreview {
 
 # Forms
 
-> TODO
+### Angular
+```js
+@Component({
```
Comment (feimosi) [inline README.md]: Could you add the necessary imports on top?

File: README.md [hunk 2]
```diff
@@ -344,7 +344,28 @@ export class UserPreview {
 
 # Forms
 
-> TODO
+### Angular
+```js
```
Comment (feimosi) [inline README.md]: Please add some short info that it's a reactive form example.

File: README.md [hunk 3]
```diff
@@ -344,7 +344,28 @@ export class UserPreview {
...
+@Component({
+  selector: 'reactive-form',
+  template: `
+    <div>
+    <form [formGroup]="form" (ngSubmit)="onSubmit(form.value, form.valid)" novalidate>
+      Name: <input type="text" formControlName="name">
```
Comment (feimosi) [inline README.md]: Would be great to add `label` here e.g.
`<label>Name: <input type="text" formControlName="name"></label>`

**Review by feimosi — CHANGES_REQUESTED** [review]
Thanks for the contribution!

File: README.md (hunk 1 above)
Comment (lvncnt) [reply README.md]: i am on it.

**Comment by lvncnt** [comment]
requested changes committed: add short info for angular reactive form, add imports and label tag

**Comment by feimosi** [comment]
Looks good, thanks!

**Pull request merged**